src/RuleErrors/DisallowedCallsRuleErrors.php

```diff
@@ -76,9 +76,14 @@ public function get(?CallLike $node, Scope $scope, string $name, ?string $displa
 
 	private function definedInMatches(DisallowedCall $disallowedCall, ?string $definedIn): bool
 	{
+		if (!$definedIn && $disallowedCall->getDefinedIn()) {
+			return false;
+		}
+
```

Comment:
Do you remember what this `if` block does? If I add
```php
require_once __DIR__ . '/libs/TestException.php';
```
to `bootstrap.php` to make the class known, that line is not covered by tests anymore.

Reply:
I have removed the `if` and let's see.